### PR TITLE
Fix: Icons are disappearing in Nested Tabs widget [ED-8844]

### PR DESCRIPTION
**Pull request by hein-obox**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js [hunk 1]
```diff
@@ -279,6 +266,33 @@ test.describe( 'Nested Tabs tests', () => {
...
+
+		// Add widgets.
+		await editor.addWidget( 'nested-tabs', container );
+		await editor.getPreviewFrame().waitForSelector( '.e-n-tab-title.e-normal.e-active' );
+
+		// Act.
+		// Add tab icons.
+		await setIconsToTabs( page, TabsIcons );
+		const activeTabSpanCount = await editor.getPreviewFrame().locator( '.e-normal.e-active span' ).count();
+		// Update first tab title.
```
Comment (davseve) [inline tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js:286]: Please add a space above the comment.

File: modules/nested-tabs/widgets/nested-tabs.php [hunk 2]
```diff
@@ -1089,39 +1089,62 @@ protected function content_template() {
...
+					'data-binding-type': 'repeater-item',
+					'data-binding-repeater-name': 'tabs',
+					'data-binding-setting': [ 'tab_icon.value' ],
+					'data-binding-index': tabCount,
+				} );
+
+				view.addRenderAttribute( tabActiveIconKey, {
+					'class': [ 'e-n-tab-icon', 'e-active' ],
+					'data-binding-type': 'repeater-item',
+					'data-binding-repeater-name': 'tabs',
```
Comment (davseve) [inline modules/nested-tabs/widgets/nested-tabs.php:1139]: I will begin by saying that this is the first time I see this templating code. 
I played with it a bit,  Do we need `class`, `data-binding-type` and `data-binding-repeater-name`?
What is their role in the data updating process?
I deleted them and it looks like they don't affect the functionality.
We can chat about that.

Same for all the `addRenderAttribute... ` above

**Review by davseve — CHANGES_REQUESTED** [review]

File: modules/nested-tabs/widgets/nested-tabs.php (hunk 2 above)
Comment (hein-obox) [reply modules/nested-tabs/widgets/nested-tabs.php:1139]: @davseve 

`Class`: generates the class for the item. We need it.

`data-binding`-arguments: create the data-binding effect. We need them to create the data-binding effect.

`type` and `name`: I assume that we need them. The `type` is referenced inside `wp-content\plugins\elementor\assets\dev\js\editor\elements\views\base.js`

`data-binding-repeater-name`: I am not sure if this argument is required.

File: tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js (hunk 1 above)
Comment (hein-obox) [reply tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js:287]: Done.

File: modules/nested-tabs/widgets/nested-tabs.php (hunk 2 above)
Comment (hein-obox) [reply modules/nested-tabs/widgets/nested-tabs.php:1139]: @davseve I have have tested it. It doesn't work well when we leave out some of the binding attributes.

**Review by davseve — DISMISSED** [review]

**Review by TzviRabinovitch — APPROVED** [review]

**Pull request merged**